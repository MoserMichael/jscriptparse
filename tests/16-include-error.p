    # check formatting of errors in functions that come from included files

    use "tests/throwfromuse.p"

    countdownAndError(5)
