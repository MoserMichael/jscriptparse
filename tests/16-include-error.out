Error: Can't divide by zero
#(tests/throwfromuse.p:13)     return 42 / 0
                         |...............^
#(tests/throwfromuse.p:12) def countdownAndError(n) {
                         |.^
#(tests/throwfromuse.p:11)         return countdownAndError(n-1)
                         |................^
#(tests/throwfromuse.p:10) def countdownAndError(n) {
                         |.^
#(tests/throwfromuse.p:9)         return countdownAndError(n-1)
                        |................^
#(tests/throwfromuse.p:8) def countdownAndError(n) {
                        |.^
#(tests/throwfromuse.p:7)         return countdownAndError(n-1)
                        |................^
#(tests/throwfromuse.p:6) def countdownAndError(n) {
                        |.^
#(tests/throwfromuse.p:5)         return countdownAndError(n-1)
                        |................^
#(tests/throwfromuse.p:4) def countdownAndError(n) {
                        |.^
#(tests/throwfromuse.p:3)         return countdownAndError(n-1)
                        |................^
#(tests/throwfromuse.p:2) def countdownAndError(n) {
                        |.^
#(tests/16-include-error.p:1)     countdownAndError(5)
                            |.....^
